src/R/Components/Impl/Settings/IRSettings.cs

```diff
@@ -1,10 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System.ComponentModel;
+using Microsoft.Common.Core.Logging;
 using Microsoft.R.Components.ConnectionManager;
 
 namespace Microsoft.R.Components.Settings {
-    public interface IRSettings {
+    public interface IRSettings: INotifyPropertyChanged {
```

Comment:
it is just System.ComponentModel, pretty basic stuff, it doesn't have much to do with WPF. 

---
In reply to: [81233986](https://github.com/Microsoft/RTVS/pull/2408#discussion_r81233986) [](ancestors = 81233986)